integration_tests/tests/withdrawals/withdrawals.cy.ts

```diff
@@ -50,12 +56,20 @@ context('Withdrawals', () => {
 
     // Then I am asked what I want to withdraw
     const newWithdrawalPage = new NewWithdrawalPage('What do you want to withdraw?')
+
+    // And I am shown the correct withdrawables
+    newWithdrawalPage.shouldShowWithdrawableTypes(['placementRequest', 'placement'])
```

Comment:
`placementRequest` here includes placement applications?

Reply:
Yeah in the withdrawables journey, for the user, they are the same thing